packages/react-cap-theme/src/components/Dialog/DialogSurface.styles.ts

```diff
@@ -0,0 +1,29 @@
+import {
+  makeStyles,
+  mergeClasses,
+  type DialogSurfaceState,
+  useDialogSurfaceStyles_unstable,
+} from '@fluentui/react-components';
+
+import { CAP_TOKENS } from '../../theme/CAPTheme';
+
+export const useDialogSurfaceStyles = makeStyles({
+  root: {
+    borderRadius: CAP_TOKENS['cap/Dialog/Corner'],
+    border: `${CAP_TOKENS['cap/Dialog/strokeWidth']} solid ${CAP_TOKENS['cap/Dialog/strokeColor']}`,
```

Comment:
@sxsx1xsxs could you update it please to use `shorthands` util, to be consisten to Fluent:
` ...shorthands.borderColor(borderPressed)`